src/utils/is-nil/isNil.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Is the value equal to undefined or null
+ * @param  {*} value
+ * @return {boolean}
+ */
+export const isNil = value => value == null;
```

Comment:
Let's not use loose equals. Maybe...

```suggestion
export const isNil = value => [undefined, null].includes(value);
```

?